#!/usr/bin/env python
PACKAGE = "moma_bringup"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("h_lower", int_t, 0, "Lower bound for Hue", 35, 0, 179)
gen.add("s_lower", int_t, 0, "Lower bound for Saturation", 50, 0, 255)
gen.add("v_lower", int_t, 0, "Lower bound for Value", 50, 0, 255)

gen.add("h_upper", int_t, 0, "Upper bound for Hue", 85, 0, 179)
gen.add("s_upper", int_t, 0, "Upper bound for Saturation", 255, 0, 255)
gen.add("v_upper", int_t, 0, "Upper bound for Value", 255, 0, 255)

exit(gen.generate(PACKAGE, "pointcloud_splitter_param", "HSV"))